pokemongo_bot/cell_workers/move_to_fort_worker.py

```diff
@@ -41,6 +41,8 @@ def work(self):
                 while distance(i2f(self.api._position_lat), i2f(self.api._position_lng), lat, lng) > 10:
                     if step_walker.step():
                         break
+                    else:
+                        self.bot.process_cells(work_on_forts=False)
```

Comment:
This looks really bad, workers should definitely not be calling methods like this on the bot. 

Especially this one. Workers should just return control to the event loop to run again. If they must, they should change a config property so they run differently next time. If this is done incorrectly, we can easily end up in an infinite loop and get a stack overflow error. This is very very scary.